src/main/java/it/gov/pagopa/payhub/activities/dto/debtposition/InstallmentIngestionFlowFileDTO.java

```diff
@@ -0,0 +1,132 @@
+
+    @CsvBindByName(column = "draft")
+    private Boolean draft;
+
+    @CsvBindByName(column = "iupdOrg")
+    private String iupdOrg;
+
+    @CsvBindByName(column = "description", required = true)
+    private String description;
+
+    @CsvCustomBindByName(column = "validityDate", converter = CsvOffsetDateTimeConverter.class)
+    private OffsetDateTime validityDate;
+
+    @CsvBindByName(column = "multiDebtor")
+    private Boolean multiDebtor;
+
+    @CsvCustomBindByName(column = "notificationDate", converter = CsvOffsetDateTimeConverter.class)
+    private OffsetDateTime notificationDate;
+
+    @CsvBindByName(column = "paymentOptionIndex", required = true)
+    private Integer paymentOptionIndex;
+
+    @CsvBindByName(column = "paymentOptionType", required = true)
+    private String paymentOptionType;
+
+    @CsvBindByName(column = "paymentOptionDescription")
+    private String paymentOptionDescription;
+
+    @CsvBindByName(column = "iud")
+    private String iud;
+
+    @CsvBindByName(column = "iuv")
+    private String iuv;
+
+    @CsvBindByName(column = "entityType", required = true)
+    private EntityTypeEnum entityType;
+
+    @CsvBindByName(column = "fiscalCode", required = true)
+    private String fiscalCode;
+
+    @CsvBindByName(column = "fullName", required = true)
+    private String fullName;
+
+    @CsvBindByName(column = "address")
+    private String address;
+
+    @CsvBindByName(column = "civic")
+    private String civic;
+
+    @CsvBindByName(column = "postalCode")
+    private String postalCode;
+
+    @CsvBindByName(column = "location")
+    private String location;
+
+    @CsvBindByName(column = "province")
+    private String province;
+
+    @CsvBindByName(column = "nation")
+    private String nation;
+
+    @CsvBindByName(column = "email")
+    private String email;
+
+    @CsvCustomBindByName(column = "dueDate", converter = CsvOffsetDateTimeConverter.class)
+    private OffsetDateTime dueDate;
+
+    @CsvBindByName(column = "amount", required = true)
+    private BigDecimal amount;
+
+    @CsvBindByName(column = "debtPositionTypeCode", required = true)
+    private String debtPositionTypeCode;
+
+    @CsvBindByName(column = "paymentTypeCode")
+    private String paymentTypeCode;
+
+    @CsvBindByName(column = "remittanceInformation", required = true)
+    private String remittanceInformation;
+
+    @CsvBindByName(column = "legacyPaymentMetadata")
+    private String legacyPaymentMetadata;
+
+    @CsvBindByName(column = "flagPagoPaPayment", required = true)
+    private Boolean flagPagoPaPayment;
+
+    @CsvBindByName(column = "balance")
+    private String balance;
+
+    @CsvBindByName(column = "flagMultiBeneficiary")
+    private Boolean flagMultiBeneficiary;
+
+    @CsvBindByName(column = "numberBeneficiary")
+    private Integer numberBeneficiary;
+
+    @CsvBindAndJoinByName(column = "*_*", elementType = String.class)
+    private MultiValuedMap<String, String> transfers;
```

Comment:
what do you think to formalize explicitly the number of possible transfers?

an approach *_* could be dangerouse

```suggestion
    @CsvBindAndJoinByName(column = "*_2", elementType = String.class)
    private MultiValuedMap<String, String> transfer2;
    
    @CsvBindAndJoinByName(column = "*_3", elementType = String.class)
    private MultiValuedMap<String, String> transfer3;
    
    @CsvBindAndJoinByName(column = "*_4", elementType = String.class)
    private MultiValuedMap<String, String> transfer4;
    
    @CsvBindAndJoinByName(column = "*_5", elementType = String.class)
    private MultiValuedMap<String, String> transfer5;
```